docs/refarch/boundedContexts/security/index.md

```diff
@@ -1,21 +1,71 @@
-# {name} BC
+# Security BC
+
+1. Overview
+2. User / Operator login (AuthN)
+3. Authorization Model (AuthZ)
+4. BC bootstrap 
+5. BC startup
+6. Role / Privilege association
+7. Example Query / call
 
-{overview}
 
 ## Terms
 
-Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
+Terms with specific and commonly accepted meaning within the Security Bounded Context.
 
-| Term | Description |
+| Crypto providers | Adapter that will provide cryptographic services and Key Management System |
 |---|---|
-| Term1 | Description1 |
+| IAM | Identity and Access Management. Adapter that will provide services for menus, profiles, roles and permissions.  |
+
+
+## Overview
+
+The protocol is request and response based, and the used transport protocol is secure Hypertext Transfer Protocol Secure (HTTPS). All services use the HTTP POST and GET methods. Both the request and response bodies are encoded in JSON-formatted text.
+
 
 ## Use Cases
 
-### Perform Transfer (universal mode)
+### User / Operator login (AuthN)
+
+The login function requires the user id and a secret key passed in the http body. The response contains a signed JWT token. The signature is generated by the Crypto subsystem.
+The login is performed by the Authorization services or IAM.
+
+![Use Case - Example REPLACE ME](./assets/securityBCv0.3.png)
```

Comment:
Please change the info text for the image to something more useful then the default template `Use Case - Example REPLACE ME`.